Tasks/PipAuthenticateV1/task.json

```diff
@@ -0,0 +1,98 @@
+{
+  "id": "5e3feff0-c5ae-11e8-a7d0-4bd3b8229800",
+  "name": "PipAuthenticate",
+  "friendlyName": "Python pip authenticate",
+  "description": "Authentication task for the pip client used for installing Python distributions",
+  "author": "Microsoft Corporation",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/package/pip-authenticate",
+  "helpMarkDown": "[Learn more about this task](https://pip.pypa.io/en/stable/reference/pip_install/) or [see the pip documentation](https://pip.pypa.io/)",
+  "category": "Package",
+  "version": {
+      "Major": 1,
+      "Minor": 0,
+      "Patch": 0
+  },
+  "runsOn": [
+      "Agent",
+      "DeploymentGroup"
+  ],
+  "minimumAgentVersion": "2.115.0",
```

Comment:
Can you change this to 2.120.0, so that we always log telemetry

Reply:
cool, will do

Reply:
I'll do this as well for mine